From 0cfb89b655b968a4b1460dd97e446d159c3786be Mon Sep 17 00:00:00 2001
From: tmueller <tmueller@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Fri, 10 Aug 2018 06:41:30 +0000
Subject: [PATCH 316/462] MANTIS 51444 BhsServer writes monitoring data to
 dedicated OPC UA servers - Handshake via Dispatch Data Acknowledgement . add
 new jars jackson-jr-all-2.9.0.jar, guava-23.3-android.jar

git-svn-id: svn://localhost/SelfServiceBuild/trunk@549 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntProperties.xml | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/SelfServiceBuild/Build/java/AntProperties.xml b/SelfServiceBuild/Build/java/AntProperties.xml
index 3758f82..32687d5 100644
--- a/SelfServiceBuild/Build/java/AntProperties.xml
+++ b/SelfServiceBuild/Build/java/AntProperties.xml
@@ -294,6 +294,8 @@
 <property name="JarJacksonAnnotations"      value="${DIR_GLOBALS}/ThirdParty/jackson/2.4.0/jackson-annotations-2.4.0.jar"/>
 <property name="JarJacksonDatabind"         value="${DIR_GLOBALS}/ThirdParty/jackson/2.4.1/jackson-databind-2.4.1.jar"/>
 <property name="JarJacksonJaxbAnnotations"  value="${DIR_GLOBALS}/ThirdParty/jackson/2.4.1/jackson-module-jaxb-annotations-2.4.1.jar"/>
+<property name="JarJacksonJrAll"            value="${DIR_GLOBALS}/ThirdParty/jackson/2.9.0/jackson-jr-all-2.9.0.jar"/>
+
 
 <!-- jaxb -->
 <property name="JarJaxbImpl"                value="${DIR_GLOBALS}/ThirdParty/jaxb/2.2.7/jaxb-impl-2.2.7.jar"/>
@@ -341,6 +343,8 @@
 <!-- xstream -->
 <property name="JarXStream"               value="${DIR_GLOBALS}/ThirdParty/xstream/1.4.6/xstream-1.4.6.jar"/>
 
+<!-- guava -->
+<property name="JarGuava"                   value="${DIR_GLOBALS}/guava/23.3-android/guava-23.3-android.jar"/>
 
 <!-- Definition des IDL-Compilers: OpenORB ======================= -->
 
-- 
2.41.0.windows.1

